v2/client.go

```diff
@@ -0,0 +1,84 @@
+package twitter
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strings"
+)
+
+const (
+	tweetMaxIDs = 100
+)
+
+// Client is used to make twitter v2 API callouts.
+type Client struct {
+	Authorizer Authorizer
+	Client     *http.Client
+	Host       string
+}
+
+// TweetLookup returns information about a tweet or group of tweets specified by a group of tweet ids.
+func (c *Client) TweetLookup(ctx context.Context, ids []string, opts TweetLookupOpts) (*TweetLookupResponse, error) {
+	ep := tweetLookupEndpoint.url(c.Host)
+	switch {
+	case len(ids) == 0:
+		return nil, fmt.Errorf("tweet lookup: an id is required: %w", ErrParameter)
+	case len(ids) > tweetMaxIDs:
```

Comment:
Do you think this should be the responsibility of the client lib?  I totally back you up if you think so...I'm kind of on the fence about it.

Reply:
I think so as it is stated in the documentation.